volume_group_types.go

```diff
@@ -39,7 +39,7 @@ type VolumeGroupCreate struct {
 	// Unique identifier of an optional protection policy to assign to the volume group.
 	ProtectionPolicyID string `json:"protection_policy_id,omitempty"`
 	// For a primary or a clone volume group, this property determines whether snapshot sets of the group will be write order consistent.
-	IsWriteOrderConsistent bool `json:"is_write_order_consistent"`
+	IsWriteOrderConsistent bool `json:"is_write_order_consistent,omitempty"`
```

Comment:
Can we specify , when empty value passed, what is the behaviour,  for easy understanding of parameter?

Reply:
Addressed